src/types/capsule.rs

```diff
@@ -5,6 +5,8 @@ use crate::{Bound, Python};
 use crate::{PyErr, PyResult};
 use std::ffi::{c_char, c_int, c_void};
 use std::ffi::{CStr, CString};
+use std::ptr::{self, NonNull};
+
 /// Represents a Python Capsule
```

Comment:
We should probably add some information here about the name checking, I will do that.